public/js/post/course_components.js

```diff
@@ -178,7 +178,7 @@ export var MultipleCourseCode = React.createClass({
                 <div id = {'spec' + this.props.courseID.substring(5, this.props.courseID.length)} className='more-info'>
                     <p className="full_name"> 
                         {Array.apply(0, Array(this.props.textBoxNumber)).map(function (x, i) {
-                            if (me.isValidExtraCourse(me.state.textboxValues[i])) {
+                            if (me.isValidExtraCourse(me.state.textboxValues[i].substring(0, 6))) {
```

Comment:
`.substring(0, 6)` is fairly ambiguous to new eyes. Perhaps defining a function could make this easier to understand? I noticed this in a recent pass over some of the current code in Courseography.